Valentin_Khokhlov/1/koans/about_dice_project.rb

```diff
@@ -0,0 +1,76 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+class DiceSet
+  attr_reader :dice
+
+  def roll(count)
+    @dice = []
+    pool = [*1..6]
+    
+    count.times do
+      @dice << pool.sample
```

Comment:
Your approach is good, but it can be simpler in such situation. Like this:
```suggestion
      @dice << rand(1..6)
```

Reply:
fixed all ur comments
sry for being so late
better late then never i guess xD